eclass/kernel-install.eclass

```diff
@@ -541,16 +544,18 @@ kernel-install_pkg_pretend() {
 
 	if ! use initramfs && ! has_version "${CATEGORY}/${PN}[-initramfs]"; then
 		ewarn
-		ewarn "WARNING: The standard configuration of the Gentoo distribution"
-		ewarn "kernels requires an initramfs! You have disabled the initramfs"
-		ewarn "USE flag and as a result dracut was not pulled in as a dependency."
-		ewarn "Please ensure that you are either overriding the standard"
-		ewarn "configuration or that an alternative initramfs generation plugin"
-		ewarn "is installed for your installkernel implementation!"
+		ewarn "WARNING: The standard Gentoo distribution kernel configuration is designed"
+		ewarn "to be used with an initramfs! Although possible, there is no guarantee"
+		ewarn "distribution kernels will boot without an initramfs."
 		ewarn
-		ewarn "This is an advanced use case, you are on your own to ensure"
-		ewarn "that your system is bootable!"
+		ewarn "The initramfs USE flag is disabled, and sys-kernel/installkernel is not"
+		ewarn "configured to use an initrd_generator, so an initramfs will not be built."
+		ewarn "Please ensure that an initramfs generator is installed, and installkernel"
+		ewarn "is configured to use it."
 		ewarn
+		ewarn "If a custom initramfs generator is used, installkernel must be"
+		ewarn "configured to use it. Please refer to the installkernel documentation:"
+		ewarn "https://wiki.gentoo.org/wiki/Installkernel#initrd_generator"
```

Comment:
I think this page might be more helpful then the installkernel page:

https://wiki.gentoo.org/wiki/Project:Distribution_Kernel#Using_custom_initramfs_and_UKI_generators

Reply:
> I think this page might be more helpful then the installkernel page (or maybe both):
> 
> https://wiki.gentoo.org/wiki/Project:Distribution_Kernel#Using_custom_initramfs_and_UKI_generators

should that page be changed to say "initrd generators other than ones integrated into portage"? I think "external" maybe describes this? There's probably better terms for systems that aren't integrated into the installkernel package/kernel-install eclass